### PR TITLE
auto-update documentation for EasyBuild v4.4.1

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: docs/version-specific/easyblocks.rst [hunk 1]
```diff
@@ -223,6 +227,7 @@
   * EB_Gctf
   * EB_libsmm
   * EB_Modeller
+  * EB_MOOSE
```
Comment (migueldiascosta) [inline docs/version-specific/easyblocks.rst:230]: ?

Comment (boegel) [reply docs/version-specific/easyblocks.rst:230]: Oh, that's a local one, totally my bad... Nice catch! Fixing it now...

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**